### PR TITLE
Cleanup and Enumification

### PR DESCRIPTION
**Pull request by tmf97**
### Summary

This PR groups the kwarg string definitions in constants.py into enums, and propagates those changes to all of FSW.
Also, some minor improvements to commands and the command handler

### Testing

No "new" code, old unit tests should suffice (famous last words)

### Notes

Is it de-jank or un-jank?

### Comments
- [x] Add an x between the brackets if you commented your code well!

### CODE REVIEW

File: utils/constants.py [hunk 1]
```diff
@@ -59,154 +59,181 @@
...
-
-T_START = "t_start"
-T_STOP = "t_stop"
-
-DECIMATION_FACTOR = "decimation_factor"
-
-INDEX = "index"
-VBATT = "vbatt"
+@unique
+class CommandKwargs(StringEnum):
```
Comment (sjz38) [inline utils/constants.py:72]: Comments to the new Enums you made wouldn't hurt

File: communications/codecs.py [hunk 2]
```diff
@@ -1,70 +1,71 @@
...
+LATCHUPS5_codec = Codec(dk.LATCHUPS5, "ushort")
+LATCHUPS6_codec = Codec(dk.LATCHUPS6, "ushort")
+WDT_TIME_LEFT_I2C_codec = Codec(dk.WDT_TIME_LEFT_I2C, "int")
+WDT_TIME_LEFT_GND_codec = Codec(dk.WDT_TIME_LEFT_GND, "int")
+WDT_COUNTS_I2C_codec = Codec(dk.WDT_COUNTS_I2C, "int")
+WDT_COUNTS_GND_codec = Codec(dk.WDT_COUNTS_GND, "int")
+GOM_BOOTS_codec = Codec(dk.GOM_BOOTS, "int")
+GOM_BOOTCAUSE_codec = Codec(dk.GOM_BOOTCAUSE, "uint8")
+GOM_BATTMODE_codec = Codec(dk.GOM_BATTMODE, "uint8")
 # beware integer underflow erro
```
Comment (sjz38) [inline communications/codecs.py:36]: erro -> error

File: communications/command_definitions.py [hunk 3]
```diff
@@ -17,6 +17,13 @@
 import drivers.power.power_structs as ps
 import time
 import hashlib
+from utils.constants import (
+    FMEnum,
+    CommandEnum,
+    CommandKwargs as ck,
+    DownlinkKwargs as dk,
+    GomConfKwargs,
```
Comment (sjz38) [inline communications/command_definitions.py:25]: Seems like you could name this gk, if we're being consistent with the other Kwarg imports

File: communications/command_definitions.py [hunk 4]
```diff
@@ -746,125 +742,125 @@ def _method(self, parent: MainSatelliteThread, **kwargs) -> None:
...
+    Codec(GomConfKwargs.OUTPUT_SAFE8, "bool"),
+    Codec(GomConfKwargs.OUTPUT_ON_DELAY, "ushort"),
+    Codec(GomConfKwargs.OUTPUT_OFF_DELAY, "ushort"),
+    Codec(GomConfKwargs.VBOOST1, "ushort"),
+    Codec(GomConfKwargs.VBOOST2, "ushort"),
+    Codec(GomConfKwargs.VBOOST3, "ushort"),
 ]
 
 
 class set_gom_conf1(Command):
```
Comment (sjz38) [inline communications/command_definitions.py:862]: remove empty line

**Review by sjz38 — CHANGES_REQUESTED** [review]
👍I'll approve after you address my comments

File: communications/command_definitions.py (hunk 3 above)
Comment (tmf97) [reply communications/command_definitions.py:25]: Yeah, but not that important

File: communications/command_definitions.py (hunk 4 above)
Comment (tmf97) [reply communications/command_definitions.py:862]: Can't :)
https://www.python.org/dev/peps/pep-0008/#blank-lines

File: utils/constants.py (hunk 1 above)
Comment (sjz38) [reply utils/constants.py:72]: Reminder ^

**Review by sjz38 — APPROVED** [review]
👍

**Pull request merged**